Source/WebCore/platform/gamepad/cocoa/GameControllerGamepadProvider.mm

```diff
@@ -39,6 +39,19 @@
 
 #import "GameControllerSoftLink.h"
 
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/GameControllerAdditions.h>)
+#import <WebKitAdditions/GameControllerAdditions.h>
```

Comment:
We actually need to include the .mm file here, to get the implementation inlined.

Reply:
(Actually the way it's been done in additions, that's not true, but carrying on further conversation on slack)